website/docs/Revocation.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: Revocation.md
+sidebar_position: 30
+---
+
+:::caution
+
+Revocation is under active development and is not currently considered stable.
+If you need Revocation immidiately, we suggest using a [Custom Revocation Contract](#custom-revocation-contract).
+
+:::
+
+## Custom Revocation Contract
+
+It is possible to implement Revocation using [Conditions](Conditions.md) that rely on a function call to a [Custom Smart Contract](Conditions.md#function-call-of-non-standard-contract).
+This allows the handling of revocation to be decentralised and transparent.
+Here is an example of the smart contract (not suitable for production):
```

Comment:
I had no idea this was a possibility. Where did that idea originate?

Reply:
revocation is no different to a condition, it's just that the condition is that no one is allowed to decrypt